mpich-EL_7.spec

```
@@ -1,7 +1,7 @@
 Summary:        A high-performance implementation of MPI
 Name:           mpich
-Version:        4.0~a1
-Release:        2.g032b3aeb2%{?dist}
+Version:        4.0~a2
+Release:        1.g65dcaccf3%{?dist}
```

Since this is the actual upstream 4.0a2 release with no other commits on top, we can drop the git hash in the `Release:` here:
```suggestion
Release:        1%{?dist}
```